content/en/docs/kubernetes/collector/components.md

```diff
@@ -13,13 +13,148 @@ enhancing it.
+and logs. It enables you to correlate your application's traces, metrics, and
+logs with other Kubernetes telemetry such as pod metrics and events.
+
+The Kubernetes Attributes Processor uses the Kubernetes API to discover all pods
+running in a cluster, keeps a record of their IP addresses, pod UIDs and
+interesting metadata. By default data passing through the processor is
+associated to a pod via the incoming request's IP address, but different rules
+can be configured. Since the processor uses the Kubernetes API it requires
+special permissions. If you're using the
+[OpenTelemetry Collector Helm chart](../../helm/collector/) you can use the
+[`kubernetesAttributes` preset](../../helm/collector/#kubernetes-attributes-preset)
+to get started.
+
+The following attributes are added by default:
+
+- k8s.namespace.name
+- k8s.pod.name
+- k8s.pod.uid
+- k8s.pod.start_time
+- k8s.deployment.name
+- k8s.node.name
+
+The Kubernetes Attributes Processor can also set resource attributes from labels
+and annotations of pods and namespaces.
+
+```yaml
+k8sattributes:
+  auth_type: 'serviceAccount'
+  extract:
+    metadata: # metadata to extra from the pod
+      - k8s.namespace.name
+      - k8s.pod.name
+      - k8s.pod.start_time
+      - k8s.pod.uid
+      - k8s.deployment.name
+      - k8s.node.name
+    annotations:
+      - tag_name: a1 # extracts value of annotation from pods with key `annotation-one` and inserts it as a tag with key `a1`
+        key: annotation-one
+        from: pod
+      - tag_name: a2 # extracts value of annotation from namespaces with key `annotation-two` with regexp and inserts it as a tag with key `a2`
+        key: annotation-two
+        regex: field=(?P<value>.+)
+        from: namespace
+    labels:
+      - tag_name: l1 # extracts value of label from namespaces with key `label1` and inserts it as a tag with key `l1`
+        key: label1
+        from: namespace
+      - tag_name: l2 # extracts value of label from pods with key `label2` with regexp and inserts it as a tag with key `l2`
+        key: label2
+        regex: field=(?P<value>.+)
+        from: pod
+  pod_association: # how to associate the data to a pod (order matters)
+    - sources: # First try to use the value of the resource attribute k8s.pod.ip
+        - from: resource_attribute
+          name: k8s.pod.ip
+    - sources: # Then try to use the value of the resource attribute k8s.pod.uid
+        - from: resource_attribute
+          name: k8s.pod.uid
+    - sources: # If neither of those work, use the request's connection to get the pod IP.
+        - from: connection
+```
+
+There are also special configuration options for when the collector is deployed
+as a Daemonset or as a Deployment. For Kubernetes Attributes Processor
+configuration details, see
+[Kubernetes Attributes Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor).
```

Comment:
```suggestion
[Kubernetes Attributes Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/k8sattributesprocessor#deployment-scenarios).
```

Probably better to link to here? Or at least have another link that's specific to deployment scenarios.

Reply:
I'm not ready to link to that doc yet.  I would prefer these docs use kubernetes terms, which unfortunately aren't compatible with the generic deployment patterns.

Reply:
Updated

Reply:
I see now that you wanted me to link to the actual section of the README, not the site's collector deployment doc.  I've adjusted this section and linked directly to the important README section.

Reply:
dank, thanks